### ISSUE TITLE
TradingView comparison for SuperTrend on Renko

### ISSUE THREAD

**Issue by kokazani** [DaveSkender/Stock.Indicators#945]
Hello, 

I think the calculation of SuperTrend on Renko quotes is not correct. Definitely is not the same as painted on TradingView.

Inputs:
- I use the Renko(50) on BTCUSDTPERP
- SuperTrend (10, 3) [Community Script by KivancOzbilgic , with hl2 calculations as skender.SuperTrend. The default by TradingView is on close]. 

[https://ibb.co/SJn4Pnd](https://ibb.co/SJn4Pnd)

* To achieve the same brick steps as TradingView, I use this 
```
-- Changing the values of first quote in the list to have a correct result on +/- 50 usdt as tradingview.
Dim brickSize As Decimal = 50 
Dim ql As List(Of Skender.Stock.Indicators.Quote) = curr_c.quoteList
ql.First().Open = (ql.First().Open \ brickSize) * brickSize
ql.First().Close = (ql.First().Close \ brickSize) * brickSize
ql.First().High = (ql.First().High \ brickSize) * brickSize
ql.First().Low = (ql.First().Low \ brickSize) * brickSize 

-- Calculate the indicators
Dim rl As List(Of Skender.Stock.Indicators.RenkoResult) = ql.GetRenko(50).ToList()
Dim stochRsi As StochRsiResult = ql.GetRenko(50).GetStochRsi(14, 14, 3, 3).Last()
Dim superTrend As List(Of SuperTrendResult) = ql.GetRenko(50).GetSuperTrend(10, 3).ToList()
```
The renko quote list is correct. The stochRsi is correct but the SuperTrend is not. I think something is wrong with ATR calculations.

Thank you,

**Comment by DaveSkender** [comment]
> I think something is wrong with ATR calculations.

Our SuperTrend uses the standard library ATR calculation.  If it were wrong, we’d see a lot of failing unit tests and have a lot of reported issues given how often it’s reused in other indicators.  You can run a straight `GetAtr(..)` on it to see those values directly.

**Comment by DaveSkender** [comment]
See if the small mod in #947, recently implemented in v2.4.1, improves your comparison.  This change only impacts warmup and convergence period values.
